src/main/kotlin/com/jetbrains/snakecharm/lang/psi/types/AbstractSmkRuleOrCheckpointType.kt

```diff
@@ -133,13 +134,11 @@ abstract class AbstractSmkRuleOrCheckpointType<T: SmkRuleOrCheckpoint>(
     override fun isBuiltin() = false
 
     companion object {
-        fun <T: SmkRuleOrCheckpoint> createRuleLikeLookupItem(name: String, elem: T): LookupElement {
+        fun <T: SmkRuleOrCheckpoint> createRuleLikeLookupItem(name: String, elem: T, currentFile: PsiFile): LookupElement {
             val containingFileName = elem.containingFile.name
             val displayPath = try {
-                val elementFilePath = elem.containingFile.originalFile.virtualFile?.presentableUrl ?: containingFileName
-                val elementFile = File(elementFilePath)
-                val module = ModuleUtilCore.findModuleForPsiElement(elem.originalElement)
-                val currentDirectory = File(module?.moduleFilePath ?: elementFilePath).parentFile
+                val elementFile = File(elem.containingFile.originalFile.virtualFile?.presentableUrl ?: containingFileName)
```

Comment:
I think `originalFile` not needed here + please keep lines not very long

Reply:
If I don't use `originalFile`, a call to `virtualFile` here returns null in completion tests in `localrules_and_ruleorder.feature` since `getViewProvider().isEventSystemEnabled()` from `PsiFileImpl` returns false. I have spent some time debugging this while working on another issue but I haven't been able to figure out why this happens (somehow only for completion, not only in tests but also in an IDEA debug instance; resolve, however, works just fine). I discovered that a call to `originalFile` solves this particular problem without having to modify anything else so I went with it. I can try to spend more time on debug but I'm not sure if it's that necessary? I assumed a call to `originalFile` is not that taxing so it's ok to use, but maybe this is not the case, and it if, of course, unclear if this really does solve the problem in every possible case.

Reply:
Update: I think it happens because `containingFile` created during completion is a copy, not the original file, and `isEventSystemEnabled()` should return false for that copy (see `CompletionInitializationUtil`, `insertDummyIdentifier()` method, line 123: the original file is copied to the `hostCopy` variable in `obtainFileCopy()` method, and in this method there is a check on line 183 that `isEventSystemEnabled()` should return false and this is intended). However, since we need access to the actual virtual file, we have to use `originalFile` and not this copy we're getting, I suppose. The file created as `hostCopy` and the file we get by calling `elem.containingFile` are the same object, I checked in debug.

Reply:
Ok, thanks for the explanation, so it is needed.